File: package/main/src/Predicate/isNullish.ts
/**
 * Checks whether a value is null or undefined
 * @param {unknown} value - The value to check
 * @returns {boolean} True if the value is null or undefined
 * @example
 * isNullish(null); // true
 * isNullish(undefined); // true
 * isNullish(0); // false
 * isNullish(""); // false
 */
export const isNullish = (value: unknown): value is null | undefined =>
  value === null || value === undefined;

File: package/umt_python/src/predicate/is_nullish.py
def is_nullish(value: object) -> bool:
    """
    Checks whether a value is None.

    In Python, the equivalent of JavaScript's null/undefined is None.

    Args:
        value: The value to check

    Returns:
        True if the value is None

    Example:
        >>> is_nullish(None)
        True
        >>> is_nullish(0)
        False
        >>> is_nullish("")
        False
    """
    return value is None
